Tests/MapboxCoreNavigationTests/BillingHandlerTests.swift

```diff
@@ -590,34 +590,36 @@ final class BillingHandlerUnitTests: TestCase {
         ])
     }
 
-    func disabled_testTripPerWaypoint() {
+    func testTripPerWaypoint() {
+        let waypointCoordinates = [
+            CLLocationCoordinate2D(latitude: 37.751748, longitude: -122.387589),
+            CLLocationCoordinate2D(latitude: 37.752397, longitude: -122.387631),
+            CLLocationCoordinate2D(latitude: 37.753186, longitude: -122.387721),
+            CLLocationCoordinate2D(latitude: 37.75405, longitude: -122.38781),
+            CLLocationCoordinate2D(latitude: 37.754817, longitude: -122.387859),
+            CLLocationCoordinate2D(latitude: 37.755594, longitude: -122.38793),
+            CLLocationCoordinate2D(latitude: 37.756574, longitude: -122.388057),
+            CLLocationCoordinate2D(latitude: 37.757531, longitude: -122.388198),
+            CLLocationCoordinate2D(latitude: 37.758628, longitude: -122.388322),
+            CLLocationCoordinate2D(latitude: 37.759682, longitude: -122.388401),
+            CLLocationCoordinate2D(latitude: 37.760872, longitude: -122.388511),
+        ]
+
+        let waypoints = waypointCoordinates.map({ Waypoint(coordinate: $0) })
+
         var routeOptions: NavigationRouteOptions {
-            let coordinates = [
-                CLLocationCoordinate2D(latitude: 37.751748, longitude: -122.387589),
-                CLLocationCoordinate2D(latitude: 37.752397, longitude: -122.387631),
-                CLLocationCoordinate2D(latitude: 37.753186, longitude: -122.387721),
-                CLLocationCoordinate2D(latitude: 37.75405, longitude: -122.38781),
-                CLLocationCoordinate2D(latitude: 37.754817, longitude: -122.387859),
-                CLLocationCoordinate2D(latitude: 37.755594, longitude: -122.38793),
-                CLLocationCoordinate2D(latitude: 37.756574, longitude: -122.388057),
-                CLLocationCoordinate2D(latitude: 37.757531, longitude: -122.388198),
-                CLLocationCoordinate2D(latitude: 37.758628, longitude: -122.388322),
-                CLLocationCoordinate2D(latitude: 37.759682, longitude: -122.388401),
-                CLLocationCoordinate2D(latitude: 37.760872, longitude: -122.388511),
-            ]
 
-            let waypoints = coordinates.map({ Waypoint(coordinate: $0) })
 
             return NavigationRouteOptions(waypoints: waypoints)
         }
         
         let route = Fixture.route(from: "route-with-10-legs", options: routeOptions)
         
         autoreleasepool {
-            let replayLocations = Fixture.generateTrace(for: route).shiftedToPresent().qualified()
```

Comment:
How can we drop `qualified()` here? Don't we need the route to be qualified for nav native?

Reply:
It works without this `qualified` function. Maybe it was needed at some point in the past or needed for some usecases?

Reply:
I will get back to this if we continue to see issues with this test.